boilerplate/ios/Podfile

```diff
@@ -27,7 +27,7 @@ target 'HelloWorld' do
     :path => config[:reactNativePath],
     # to enable hermes on iOS, change `false` to `true` and then install pods
     :production => production,
-    :hermes_enabled => flags[:hermes_enabled],
+    :hermes_enabled => true,
```

Comment:
Does `flags[:hermes_enabled]` not work here?

Reply:
AFAIK enabling the new arch flag will turn everything on - https://github.com/react-native-community/upgrade-support/issues/193#issuecomment-1158371500

I'm not familiar enough with pod files, is there a better way to pass this along on the pod install command line?

Reply:
I don't know, tbh. I'm okay with this, maybe with a comment saying we might want to move back to `flags[:hermes_enabled]` in the future?